pkgs/by-name/wa/wayidle/package.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, rustPlatform
+, fetchFromSourcehut
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wayidle";
+  version = "0.1.1";
+
+  src = fetchFromSourcehut {
+    owner = "~whynothugo";
+    repo = "wayidle";
+    rev = "v${version}";
+    hash = "sha256-6wULrwGnXLdrX/THanJThbykKjNKpGukw9dj0jX0/dM=";
+  };
+
+  cargoHash = "sha256-zF2s3XSXnN7jVtv/0axzHiIJd/cb6wMYAOQILXp1U5U=";
+
+  meta = with lib; {
+    description = "Wayidle waits until the Wayland compositors reports being N seconds idle.";
```

```suggestion
    description = "Execute a program when a Wayland compositor reports being N seconds idle";
```
Descriptions should not end in `.` nor should they include the package name, as per https://nixos.org/manual/nixpkgs/stable/#var-meta-description